core/src/main/java/org/infinispan/configuration/global/ThreadFactoryConfigurationBuilder.java

```diff
@@ -45,15 +53,23 @@ public ThreadFactoryConfigurationBuilder nodeName(String nodeName) {
       return this;
    }
 
-   public String name() {
+   /**
+    * @deprecated Please use {@link #threadGroupName()} instead
+    */
+   @Deprecated
+   public ThreadGroup group() {
+      throw new UnsupportedOperationException();
+   }
+
+   public String threadGroupName() {
```

Comment:
Shouldn't this method return `attributes.attribute(GROUP).get()`?

Reply:
Actually it was supposed to be named `name` instead. But I can just remove the unsupported operation methods instead , as that is cleaner.